templates/post/post.css

```diff
@@ -0,0 +1,208 @@
+/* main post styling */
+.post {
+  background-color: var(--color-off-white);
+}
+.post main {
+  background-color: var(--color-white);
+  margin:0 auto;
+  max-width: 460px;
```

Comment:
I don't really like setting a max-width on main itself. I'm curious why you needed to do this.

Reply:
Agreed, but their design has a minor nuance with background colors.  Trying to account for the gray border around main at the small break point.  I'll show you when we meet